docs/content/guides/author-apps/portable-resources/howto-author-portable-resources/index.md

```diff
@@ -0,0 +1,84 @@
+---
+type: docs
+title: "How-To: Author portable resources"
+linkTitle: "Author portable resources"
+description: "Learn how to author portable resources in Radius"
+weight: 200
+categories: "How-To"
+tags: ["portability"]
+---
+
+This guide will teach you how to author a portable resource for your [Radius Application]({{< ref "/guides/author-apps/application/overview" >}}).
+
+## Prerequisites
+
+Before you get started, you'll need to make sure you have the following tools and resources:
+- [rad CLI]({{< ref "installation#step-1-install-the-rad-cli" >}})
+- [Radius environment]({{< ref "installation#step-3-initialize-radius" >}})
+- [Radius Bicep VSCode extension]({{< ref "installation#step-2-install-the-vs-code-extension" >}})
+
+## Step 1: Choose how to define your portable resource
+
+Portable resources provide **abstraction** and **portability** to Radius Applications. Radius currently offers options to manually provision the resources or automatically provision them via Recipes.
+
+{{< tabs Recipe Manual >}}
+
+{{% codetab %}}
+
+Recipes enable a separation of concerns between infrastructure operators and developers by automating infrastructure deployment. You can run a default recipe registered in your environment or select the specific Recipe you want to run. 
+
+{{< rad file="snippets/app-redis-recipe.bicep" embed=true marker="//Recipe" >}}
+
+ To learn more visit the [Recipes overview]({{< ref "/guides/recipes/overview" >}}).
+
+{{% /codetab %}}
+
+{{% codetab %}}
+
+Add a RedisCache resource with the `resourceProvisioning` mode as `manual`. This enables you to configure the underlying Redis resource
+
+{{< rad file="snippets/app-redis-manual.bicep" embed=true marker="//MANUAL" >}}
+
+Refer to the [Redis resource schema]({{< ref "reference/resource-schema/cache/redis" >}}) for more details.
+
+{{% /codetab %}}
+
+{{< /tabs >}}
+
+## Step 2: Define a container resource
+
+In your Bicep file `app.bicep`, add a container resource that will be leveraged by your portable resource later:
+
+{{< rad file="snippets/app.bicep" embed=true marker="//EMPTYCONTAINER" >}}
+
+## Step 3: Access your portable resources
+
+You can access the portable resource via [`connections`]({{< ref "guides/author-apps/containers#connections" >}}). Update your container definition to add a connection to the new Redis cache. This results in environment variables for connection information automatically set on the container.
```

Comment:
Could you update this to show both the connections option, as well as manually acceessing properties and setting environment variables? Would be useful for developers writing new apps where they can access the environment variables we set (CONNECTION_REDIS_URL) as well as set values to existing environment variables (CACHE_URL).